### PR TITLE
MEN-5647 - deployment creation sliding in

### PR DESCRIPTION
**Pull request by mzedel**
Screenshot 2022-08-26 at 17 03 06
Screenshot 2022-08-26 at 17 03 44

### CODE REVIEW

**Comment by tranchitella** [comment]
It behaves in a strange way with the links to the "features page", see screenshot:

image

**Comment by tranchitella** [comment]
Also, after I create a deployment, if I try creating a new one (open source flavour) the button "CREATE DEPLOYMENT" stays disabled despite the selection of the group (All devices) and the artifact. A full refresh fixes it, it must be something in the state.

image

**Comment by mzedel** [comment]
Thanks @tranchitella for taking a look!

> It behaves in a strange way with the links to the "features page"

should be adressed with https://github.com/mendersoftware/gui/pull/3030/commits/72c5554fefd924be457ec8692a429b89029e9375

whereas:

> ... creating a new one (open source flavour) the button "CREATE DEPLOYMENT" stays disabled despite the selection of the group (All devices) and the artifact ...

should be addressed with [this change](https://github.com/mendersoftware/gui/compare/f340e3c8c31922721b257a62dce927ad25f446c5..72c5554fefd924be457ec8692a429b89029e9375#diff-47a71389c5ea3672d7716404673e07f06a41b4e96662a85210744a253efb8a08R192) to the multiple-deployment-creation-prevention-mechanism™️

File: src/js/components/deployments/createdeployment.js [hunk 1]
```diff
@@ -160,75 +189,114 @@ export const CreateDialog = props => {
...
-          filters={deploymentSettings.filterId ? groups[deploymentObject.group || group].filters : undefined}
-          hasNewRetryDefault={hasNewRetryDefault}
-          onSaveRetriesSetting={onSaveRetriesSetting}
-          releaseSelectionLocked={releaseSelectionLocked}
-          setDeploymentSettings={setDeploymentObject}
-        />
-      </DialogContent>
-      <DialogActions className="margin-left margin-right">
+    <Drawer anchor="right" open={open} onClose={closeWizard} PaperProps={{ style: { minWidth: '50vw' } }}>
+      <div className="flexbox margin-bottom space-between center-aligned">
```
Comment (michaelatmender) [inline src/js/components/deployments/createdeployment.js:253]: I think "margin-bottom" is not needed here

**Review by michaelatmender — CHANGES_REQUESTED** [review]
Just two small things I noticed: 

1. The header margin I left a comment on,
2. The 'retry' dropdown should be updated to match the new label text. It's no longer 'number of retries', but total number of attempts - so it should start at "1" by default This is to make it consistent with the rest of the UI

**Comment by mzedel** [comment]
> 1. The header margin I left a comment on,
> 2. The 'retry' dropdown should be updated to match the new label text. It's no longer 'number of retries', but total number of attempts - so it should start at "1" by default This is to make it consistent with the rest of the UI

Thanks @michaelatmender - these should now be adjusted 👌!

**Review by tranchitella — APPROVED** [review]
It looks good now, thank you @mzedel!

**Pull request merged**